symfony/plugins/orangehrmPerformancePlugin/test/Api/PerformanceTrackerAPITest.php

```diff
@@ -0,0 +1,109 @@
+    }
+
+    /**
+     * @dataProvider dataProviderForTestGetAll
+     */
+    public function testGetAll(TestCaseParams $testCaseParams): void
+    {
+        $this->populateFixtures('PerformanceTracker2.yml');
+        $this->createKernelWithMockServices([Services::AUTH_USER => $this->getMockAuthUser($testCaseParams)]);
+        $this->registerServices($testCaseParams);
+        $this->registerMockDateTimeHelper($testCaseParams);
+        $api=$this->getApiEndpointMock(PerformanceTrackerAPI::class,$testCaseParams);
+        $this->assertValidTestCase($api, 'getAll', $testCaseParams);
+    }
+
+    public function dataProviderForTestGetAll(): array
+    {
+        return $this->getTestCases('PerformanceTrackerAPITest.yaml', 'GetAll');
+    }
+
+    /**
+     * @dataProvider dataProviderForTestCreate
+    */
+    public function testCreate(TestCaseParams $testCaseParams): void
+    {
+        $this->populateFixtures('PerformanceTracker2.yml');
+        $this->createKernelWithMockServices([Services::AUTH_USER => $this->getMockAuthUser($testCaseParams)]);
+        $this->registerServices($testCaseParams);
+        $this->registerMockDateTimeHelper($testCaseParams);
+        $api=$this->getApiEndpointMock(PerformanceTrackerAPI::class,$testCaseParams);
+        $this->assertValidTestCase($api, 'create', $testCaseParams);
+    }
+
+    public function dataProviderForTestCreate(): array
+    {
+        return $this->getTestCases('PerformanceTrackerAPITest.yaml', 'Create');
+    }
+
+    /**
+     * @dataProvider dataProviderForTestUpdate
+    */
+    public function testUpdate(TestCaseParams $testCaseParams): void
+    {
+        $this->populateFixtures('PerformanceTracker2.yml');
+        $this->createKernelWithMockServices([Services::AUTH_USER => $this->getMockAuthUser($testCaseParams)]);
+        $this->registerServices($testCaseParams);
+        $this->registerMockDateTimeHelper($testCaseParams);
+        $api=$this->getApiEndpointMock(PerformanceTrackerAPI::class,$testCaseParams);
+        $this->assertValidTestCase($api, 'update', $testCaseParams);
+    }
+
+    public function dataProviderForTestUpdate(): array
+    {
+        return $this->getTestCases('PerformanceTrackerAPITest.yaml', 'Update');
+    }
+
+    /**
+     *@dataProvider dataProviderForTestDelete
+     */
+    public function testDelete(TestCaseParams $testCaseParams): void
+    {
+        $this->populateFixtures('PerformanceTracker2.yml');
+        $this->createKernelWithMockServices([Services::AUTH_USER => $this->getMockAuthUser($testCaseParams)]);
+        $this->registerServices($testCaseParams);
+        $this->registerMockDateTimeHelper($testCaseParams);
+        $api=$this->getApiEndpointMock(PerformanceTrackerAPI::class,$testCaseParams);
+        $this->assertValidTestCase($api, 'delete', $testCaseParams);
+    }
+
+    public function dataProviderForTestDelete(): array
+    {
+        return $this->getTestCases('PerformanceTrackerAPITest.yaml', 'Delete');
+    }
+
+
+
```

Comment:
Remove extra blank lines and add new line to end of file

Reply:
DONE